src/breadcrumb/src/BreadcrumbItem.tsx

```diff
@@ -13,7 +22,7 @@ ExtractPropTypes<typeof breadcrumbItemProps>
 export default defineComponent({
   name: 'BreadcrumbItem',
   props: breadcrumbItemProps,
-  setup (props, { slots }) {
+  setup ({ separator, href, isCurrent = false }, { slots }) {
```

Comment:
You can't do destructing in setup since the data would lost reactivity.

Reply:
For example if separator changes the item won't rerender.

Reply:
Ok! I'm new to vue so I might make some dumb mistake like this :)

Reply:
No problem, you'll be more and more experienced.